client/src/app/site/polls/components/base-poll-detail.component.ts

```diff
@@ -240,9 +250,43 @@ export abstract class BasePollDetailComponent extends BaseModelContextComponent
                         this.createVotesData();
                         this.optionsLoaded.resolve();
                         this.cd.markForCheck();
+                        this.setEntitledUsersData();
                     }
                 })
             );
         }
     }
+
+    private setEntitledUsersData(): void {
+        if (this.entitledUsersSubscription) {
+            this.entitledUsersSubscription.unsubscribe();
+        }
+        const userIds = new Set<number>();
+        for (const entry of this.poll.entitled_users_at_stop) {
+            userIds.add(entry.user_id);
+            if (entry.vote_delegated_to_id) {
+                userIds.add(entry.vote_delegated_to_id);
+            }
+        }
+        this.entitledUsersSubscription = this.userRepo
+            .getViewModelListObservable()
+            .pipe(
+                filter(users => !!users.length),
+                map(users => users.filter(user => userIds.has(user.id)))
+            )
+            .subscribe(users => {
+                const entries = [];
+                for (const entry of this.poll.entitled_users_at_stop) {
+                    entries.push({
+                        ...entry,
+                        user: users.find(user => user.id === entry.user_id),
+                        voted_verbose: `voted:${entry.voted}`,
+                        vote_delegated_to: entry.vote_delegated_to_id
+                            ? users.find(user => user.id === entry.vote_delegated_to_id)
+                            : null
+                    });
+                }
+                this.entitledUsersObservable = from([entries]);
+            });
```

Comment:
The console logs the following error:
`entitled_users_at_stop` is undefined (at `base-poll-detail.component.ts` line 265) (Error: `TypeError: "this.poll.entitled_users_at_stop" is not iterable`)